mbed/mbed.py

```diff
@@ -3461,10 +3461,6 @@ def main():
         if very_verbose:
             traceback.print_exc(file=sys.stdout)
         error("Unknown Error: %s" % e, 255)
-    finally:
```

Comment:
This was a strange thing to do in the first case, definitely good to remove it as it was generated even if there weren't any exceptions.